ios/RNFingerprintjsPro.swift

```diff
@@ -16,13 +16,21 @@ class RNFingerprintjsPro: NSObject {
     }
 
     @objc(init:region:endpoint:)
-    public required init(_ apiToken: String, _ region: String? = "us", _ endpoint: URL? = nil) {
-        fpjsClient = FingerprintJSProFactory
-            .getInstance(
-                token: apiToken,
-                endpoint: endpoint,
-                region: region
-            )
+    public required init(_ apiToken: String, _ region: String? = "us", _ endpoint: String? = nil) {
+        if (endpoint != nil) {
```

Comment:
```suggestion
        if let endpoint = endpoint, endpointURL = URL(endpoint) {
```

would save you the clumsy coalescing operator down below

you would then use the `endpointURL` in the getInstance call instead